common-ui/src/main/res/values/widgets.xml

```diff
@@ -158,14 +158,56 @@
     </style>
 
     <!-- Buttons -->
+    <style name="Widget.DuckDuckGo.DaxButton.Contained" parent="Widget.MaterialComponents.Button">
+        <item name="android:textAppearance">?attr/textAppearanceButton</item>
+        <item name="shapeAppearance">?shapeAppearanceSmallComponent</item>
+    </style>
```

Comment:
I think this style is not used anywhere

Reply:
This is the parent style for `Widget.DuckDuckGo.DaxButton.Contained.Primary` and `Widget.DuckDuckGo.DaxButton.Contained.Destructive`. Both `Primary `and `Destructive` button types use the textAppearance and shape, so we have a parent style shared.